clerk/organizations.go

```diff
@@ -214,3 +214,37 @@ func (s *OrganizationsService) ListMemberships(params ListOrganizationMembership
 	}
 	return membershipsResponse, nil
 }
+
+type CreateOrganizationMembershipParams struct {
+	OrganizationID string
+	UserID         string `json:"user_id" form:"user_id"`
+	Role           string `json:"role" form:"role"`
+}
+
+func (s *OrganizationsService) CreateMembership(params CreateOrganizationMembershipParams) (*OrganizationMembership, error) {
+	req, _ := s.client.NewRequest(http.MethodPost, fmt.Sprintf("%s/%s/memberships", OrganizationsUrl, params.OrganizationID), &params)
+
+	var organizationMembership OrganizationMembership
+	_, err := s.client.Do(req, &organizationMembership)
+	if err != nil {
+		return nil, err
+	}
+	return &organizationMembership, nil
+}
+
+type UpdateOrganizationMembership struct {
+	OrganizationID string
+	UserID         string
+	Role           string `json:"role"`
+}
+
+func (s *OrganizationsService) UpdateMembership(params UpdateOrganizationMembership) (*OrganizationMembership, error) {
```

Comment:
:wrench: Let's change the signature here as well to be in line with the rest of our methods. I don't think it's necessarily better, but at least it follows a common pattern. :/ 

```go
type UpdateOrganizationMembershipParams struct {
  Role string `json:"role"`
}

func (s *OrganizationsService) UpdateMembership(organizationID, userID string, params UpdateOrganizationMembershipParams) (*OrganizationMembership, error) {}
```

Reply:
@gkats I got as reference the `CreateInvitation()` which was the last one added on the organizations, in that function the organizationId is being retrieved from `params`. 
Shall i change this one as well?

Reply:
:disappointed: Thanks for the reference. Looks like we're all over the place. 

Let's ask for more opinions on this one. cc @clerkinc/backend-team.